### PR TITLE
Validate document nodes before use

### LINKED ISSUE #492: document.body is null in Popup._getSiteColor
Seen it happen when Google is redirecting to a website when clicking a search result. Could just make the function return some value when `document.body` is `null`, but maybe this is a timing issue that could have worse consequences. Firefox 75 / Linux

```
Yomichan v20.4.18.0 has encountered a problem.
Originating URL: https://www.google.com/url?whatever
TypeError: Window.getComputedStyle: Argument 1 is not an object.
_getSiteColor@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/popup.js:365:59
updateTheme@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/popup.js:175:58
setOptionsContext@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/popup.js:89:14
async*updateOptions@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/frontend.js:159:26
async*prepare@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/frontend.js:65:24
applyOptions@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/content-script-main.js:129:47
async*@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/content-script-main.js:147:11
async*@moz-extension://689faae7-541f-45a7-a011-773bbfe31bad/fg/js/content-script-main.js:148:3

Issues can be reported at https://github.com/FooSoft/yomichan/issues
```

### PR DESCRIPTION
**Pull request by toasted-nutbread**
Fixes #492

### CODE REVIEW

**Comment by siikamiika** [comment]
Could this issue be solved by awaiting a `load` event in `content-script-main.js`?

**Comment by toasted-nutbread** [comment]
It could be, but technically it's still possible for the body/documentElement nodes to be removed. So this change should at least handle the fringe cases.

File: ext/mixed/js/dynamic-loader.js [hunk 1]
```diff
@@ -43,12 +48,11 @@ const dynamicLoader = (() => {
...
             }
 
-            loadScriptSentinel(resolve);
+            loadScriptSentinel(parent, resolve, reject);
         });
     }
 
-    function loadScriptSentinel(resolve, reject) {
-        const parent = document.body;
+    function loadScriptSentinel(parent, resolve, reject) {
```
Comment (siikamiika) [inline ext/mixed/js/dynamic-loader.js:55]: It seem there was a bug earlier :smile:

Comment (toasted-nutbread) [reply ext/mixed/js/dynamic-loader.js:55]: 😗

**Pull request merged**